### PR TITLE
Add `.multiple()` method

### LINKED ISSUE #3: Add `.multiple()`
Allow taking multiple random items like `_.sampleSize()`:

```js
const randomItem = require('random-item');

randomItem.multiple([1, 2, 3, 4, 5], 2);
//=> [1, 4]
```

Not entirely sure on what the function name should be; maybe `.some()`.

// @sindresorhus

### PR DESCRIPTION
**Pull request by Richienb**
Fixes: #3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,3 +7,21 @@ module.exports = array => {
...
+		throw new TypeError('Expected a non-negative integer');
+	}
+
+	if (count > array.length) {
+		throw new TypeError(`Expected an array with at least ${count} items`);
+	}
+
+	array = [...array];
+
+	return [...new Array(count)].map(() => array.splice(Math.floor(Math.random() * array.length), 1)[0]);
```
Comment (sindresorhus) [inline index.js]: Seems like you can do this without splicing (which is very slow).

File: index.js [hunk 2]
```diff
@@ -7,3 +7,21 @@ module.exports = array => {
...
+	if (!Array.isArray(array)) {
+		throw new TypeError('Expected an array');
+	}
+
+	if (!Number.isInteger(count) && count >= 0) {
+		throw new TypeError('Expected a non-negative integer');
+	}
+
+	if (count > array.length) {
+		throw new TypeError(`Expected an array with at least ${count} items`);
```
Comment (sindresorhus) [inline index.js]: This should be allowed.

File: index.js (hunk 1 above)
Comment (sindresorhus) [reply index.js]: Any reason you're not just calling `module.exports()` multiple times?

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Richienb) [reply index.js]: @sindresorhus Which type of behavior do you want?

Allowing duplicates:

```js
randomItem.multiple([1, 2, 3], 2)
//=> [1, 1]
```

Disallowing duplicates:

```js
randomItem.multiple([1, 2, 3], 2)
//=> [2, 1]
```

Comment (sindresorhus) [reply index.js]: Allowing duplicates. That would be consistent with the main method. There could potentially be an option to ensure the elements are consecutively unique, and one that ensures the array uses up all elements before starting again.

Comment (sindresorhus) [reply index.js]: The behavior should be clearly documented.

**Comment by sindresorhus** [comment]
Any thoughts on https://github.com/sindresorhus/random-item/pull/4#discussion_r511670522 ?

**Comment by Richienb** [comment]
@sindresorhus We could add `.multiple` for https://github.com/sindresorhus/unique-random

**Pull request merged**